src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -68,20 +89,30 @@ const AuditLogsTable = ({ showModel = false }: Props) => {
     (column) => showModel || column.accessor !== "model"
   );
 
-  const [, setQueryParams] = useQueryParams<{ page: string | null }>({
+  const [limit, setLimit] = useState<number>(DEFAULT_LIMIT_VALUE);
+  const [{ page }, setQueryParams] = useQueryParams<{ page: string | null }>({
     page: null,
   });
```

Comment:
It could be nice to convert the page to a number using a const here instead of having to do `Number(page)` in a few places.
`const page = Number(queryParams.page);`